### PR TITLE
Adjust load test to larger clusters

### PR DESCRIPTION
**Pull request by wojtek-t**
Ref #20638

### CODE REVIEW

File: test/e2e/load.go [hunk 1]
```diff
@@ -70,7 +70,17 @@ var _ = Describe("Load capacity", func() {
 	framework.NamespaceDeletionTimeout = time.Hour
 
 	BeforeEach(func() {
-		c = framework.Client
+		// Explicitly create a client with higher QPS limits.
+		config, err := loadConfig()
+		Expect(err).NotTo(HaveOccurred())
```
Comment (timothysc) [inline test/e2e/load.go]: Could we take this out to a generic f(n) - I'm guessing it would be useful in other testing contexts.

Comment (wojtek-t) [inline test/e2e/load.go]: This is still a test PR, not ready for review.
But sure - I can do it.

**Comment by wojtek-t** [comment]
@gmarek - PTAL

**Comment by wojtek-t** [comment]
I don't understand what's happened:
Ran 120 of 226 Specs in 398.990 seconds
SUCCESS! -- 120 Passed | 0 Failed | 0 Pending | 106 Skipped 

@k8s-bot e2e test this please github issue: #IGNORE

**Pull request merged**